### PR TITLE
Reset the list wizard list-style

### PR DESCRIPTION
**Pull request by aschempp**
Fixes https://github.com/contao/contao/issues/5616

### CODE REVIEW

**Review by Toflar — DISMISSED** [review]

**Review by fritzmg — DISMISSED** [review]

File: core-bundle/contao/themes/flexible/main.css [hunk 1]
```diff
@@ -1867,6 +1867,12 @@ ul.sgallery li {
 }
 
 /* List wizard */
+.tl_listwizard, .tl_listwizard li {
```
Comment (leofeyer) [inline core-bundle/contao/themes/flexible/main.css]:
```suggestion
.tl_listwizard {
```

No need to apply this to the `li` elements, too, is there?

Comment (fritzmg) [reply core-bundle/contao/themes/flexible/main.css]: The margin and padding needs to be reset on the `li` elements.

Comment (leofeyer) [reply core-bundle/contao/themes/flexible/main.css]: Not on my system (Safari 16.2 on macOS Ventura).



In which browser on which OS does the issue occur for you?

Comment (fritzmg) [reply core-bundle/contao/themes/flexible/main.css]: You are right, I must have remembered that wrong.

**Review by leofeyer — APPROVED** [review]

**Pull request merged**